content/v2.0/monitor-alert/templates/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Monitor with templates
+seotitle: Use templates to easily start monitoring data in many supporting environments, including AWS Cloudwatch, Redis, Apache, Nginx, and more.
+description: >
+  Use templates to easily start monitoring data in many supporting environments, including AWS Cloudwatch, Redis, Apache, Nginx, and more.
```

Comment:
Use InfluxDB to monitor services like AWS, Redis, Apache, Nginx, and more. Build your own custom monitoring solution or use InfluxDB templates to quickly spin up robust monitoring solutions.

Reply:
This would be perfect for our "solutions" index page. Does it make sense to add that level now being we only offer "templates"...may look odd in the TOC unless we hide it?